onward/app/weather/models/Weather.scala

```diff
@@ -0,0 +1,23 @@
+package weather.models
+
+import play.api.libs.json.{Json, Writes}
+
+object Weather {
+  implicit val jsonReads = Json.reads[Weather]
+
+  implicit val writes = new Writes[Weather] {
+    def writes(model: Weather) = {
```

Comment:
Just another Q on whether the default weather writes would work?

Reply:
same as above, if it works without happy to remove :D